src/lib/linux-api/src/signal.rs

```diff
@@ -942,10 +947,39 @@ pub enum SignalHandler {
 #[allow(non_camel_case_types)]
 pub type linux_sigaction = bindings::linux_sigaction;
 
+/// Restorer suitable for use with `sigaction`.
+///
+/// Normally libc's implementation of the `sigaction` function injects a similar
+/// restorer function.
+///
+/// # Safety
+///
+/// This function is only intended for use as a `restorer` in `sigaction`.
+/// Do not call this function directly.
+//
+// TODO: use the language-provided `naked` attribute once stabilized.
```

Comment:
nit: link to the rfc or stabilization PR to make it easier to check if it's been stabilized?

Reply:
yeah, I'd tried chasing it down and sort of gave up. I added a comment pointing to https://github.com/rust-lang/rfcs/blob/master/text/1201-naked-fns.md as a starting point, but the status is unclear.